src/main/java/net/minecraftforge/client/IRenderHandler.java

```diff
@@ -19,14 +19,27 @@
 
 package net.minecraftforge.client;
 
+import com.mojang.blaze3d.matrix.MatrixStack;
 import net.minecraftforge.api.distmarker.Dist;
 import net.minecraft.client.Minecraft;
 import net.minecraft.client.world.ClientWorld;
 import net.minecraftforge.api.distmarker.OnlyIn;
 
+import javax.annotation.Nullable;
+
+// TODO 1.16 Split this apart for the various uses instead of stuffing everything into one interface.
```

Comment:
Why not just do that now and make this class as deprecated, and the children extending it?